examples/bankaccount/aggregate/wasmcloud.toml

```diff
@@ -1,4 +1,4 @@
-name = "Bank Account Aggregate"
+name = "bankaccount_aggregate"
```

Comment:
I fixed this with the latest version of cosmo, but I'm OK with it like this and we can confirm that later

Reply:
It's fine leaving it this way